src/scirpy/pl/_logoplots.py

```diff
@@ -0,0 +1,91 @@
+    to_type: Sequence[Literal["information", "counts", "probability", "weight"]] = "information",
+    pseudocount: float = 0,
+    background=None,
+    center_weights: bool = False,
+    plot_default=True,
+    **kwargs,
+):
+    """
+    Generates logoplots of CDR3 sequences
+
+    This is a user friendly wrapper function around the logomaker python package.
+    Enables the analysis of potential amino acid motifs by displaying logo plots.
+    Subsetting of AnnData/MuData has to be performed manually beforehand (or while calling) and only cells with equal cdr3 sequence lengths are permitted.
+
+    Parameters
+    ----------
+    {adata}
+    chains
+        One or up to two chains from which to use CDR3 sequences i.e. primary and/or secondary VJ/VDJ chains. Mixing VJ and VDJ chains will likely not lead to a meaningful result.
+    {airr_mod}
+    {airr_key}
+    {chain_idx_key}
+    cdr3_col
+        Key inside awkward array to retrieve junction information (should be in aa)
+    to_type
+        Choose one of matrix types as defined by logomaker:
+        * `"information"`
+        * `"counts"`
+        * `"probability"`
+        * `"weight"`
+    pseudocount
+        Pseudocount to use when converting from counts to probabilities
+    background
+        Background probabilities. Both arrays with the same length as ouput or df with same shape as ouput are permitted.
+    center_weights
+        Whether to subtract the mean of each row, but only if to_type == `weight`
+    plot_default
+        If true does some basic formatting for the user i.e:
+        * `"font_name"` = 'Arial Rounded MT Bold'
+        * `"color_scheme"` = 'chemistry'
+        * `"vpad"`= .05
+        * `"width"` = .9
+        And some additional styling. If false, the user needs to adapt`**kwargs` accordingly.
+    **kwargs
+        Additional arguments passed to logomaker.Logo() for comprehensive customization. For a full list of parameters please refer to logomaker documentation (https://logomaker.readthedocs.io/en/latest/implementation.html#logo-class)
+
+    Returns
+    -------
+    Returns a object of class logomaker.Logo (see here for more information https://logomaker.readthedocs.io/en/latest/implementation.html#matrix-functions)
+    """
+    params = DataHandler(adata, airr_mod, airr_key, chain_idx_key)
+    # make sure that sequences are prealigned i.e. they need to have the the same length
+    airr_df = get_airr(params, [cdr3_col], chains)
+    sequence_list = []
+    for chain in chains:
+        for sequence in airr_df[chain + "_" + cdr3_col]:
+            if sequence is not None:
+                sequence_list.append(sequence)
+
+    motif = alignment_to_matrix(
+        sequence_list, to_type=to_type, pseudocount=pseudocount, background=background, center_weights=center_weights
+    )
+    if plot_default:
+        cdr3_logo = Logo(
+            motif, font_name="Arial Rounded MT Bold", color_scheme="chemistry", vpad=0.05, width=0.9, **kwargs
```

Comment:
No, there was no particular reason why I choose this font besides for aesthetic...didn't think that something like that would matter 🫨 
For now, I just removed the argument as I couldn't find out how exactly logomaker is inferring its default font-setting....maybe we should just point out in the function description that there are many different fonts? If you ran logomaker.list_font_names() one would see all possible fonts from which one could choose...is there an elegant way to combine it maybe with the function call like say a prompt so that the user has to decide a font while calling or something similar?

Reply:
> `logomaker.list_font_names()`

That's great, wasn't aware of this! I'd just stick with the default font and mention it in the documentation.